packages/modules/web_themes/colors/source/src/assets/js/mqttClient.ts

```diff
@@ -92,7 +92,7 @@ export async function mqttPublish(topic: string, message: string) {
 		retries += 1
 	}
 	// console.warn ('MQTT publish: Now connected')
-	if (retries < 10) {
+	if (retries < 20) {
```

Comment:
Müsste nicht auch in Zeile 88 von `10` auf `20` geändert werden?

Reply:
Danke, das hatte ich übersehen. Ist jetzt gefixt und eingereicht.